.test_director

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xeuo pipefail
+
+NEW_TEST=$((git diff --name-only master | grep 'tests/.*/main.yml') || true)
```

Comment:
This can end up being more than a single test though, right? Should we run this in a loop instead? (Though that would require all tests to make sure to cleanly restore the machine to a clean state after they're done). Or maybe it's safer to only ever target a single test and error out if multiple tests are modified to force multiple PRs to be created. Not sure.